assets/airports/kmht.json

```diff
@@ -0,0 +1,976 @@
+        "SHOWZ":["N42d48m19.530", "W71d34m51.650"],
+        "SITLE":["N42d57m55.390", "W71d27m12.740"],
+        "SMYTH":["N42d30m16.560", "W73d53m05.860"],
+        "SWEDE":["N42d28m40.120", "W74d40m08.110"],
+        "TEETO":["N42d48m47.150", "W71d42m27.410"],
+        "TYLIR":["N43d03m32.930", "W71d10m01.620"],
+        "WAVBO":["N43d03m33.726", "W71d16m39.454"],
+        "WOBLY":["N42d45m30.080", "W73d29m55.810"],
+        "XOHWI":["N42d53m29.660", "W71d24m35.710"],
+        "YANTC":["N41d33m22.810", "W71d59m56.950"],
+        "YOULL":["N42d48m47.080", "W71d21m49.170"]
+    },
+    "runways":[
+        {
+            "name": ["6", "24"],
+            "end": [
+                ["N42d55m40.139", "W71d26m58.814", "221ft"],
+                ["N42d56m37.040", "W71d25m47.849", "242ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["17", "35"],
+            "end": [
+                ["N42d56m32.190", "W71d26m23.350", "216ft"],
+                ["N42d55m04.670", "W71d25m31.837", "266ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "MHT1": {
+            "icao": "MHT1",
+            "name": "Manchester One",
+            "rwy": {
+                "KMHT6" : ["#039"],
+                "KMHT24": ["#237"],
+                "KMHT17": ["#172"],
+                "KMHT35": ["#352"]
+            },
+            "body": [],
+            "exitPoints": {
+                "BOS": ["BOS"],
+                "CAM": ["CAM"],
+                "CON": ["CON"],
+                "ENE": ["ENE"],
+                "GDM": ["GDM"]
+            },
+            "draw": [
+                ["BOS*"],
+                ["CAM*"],
+                ["CON*"],
+                ["ENE*"],
+                ["GDM*"]
+            ]
+        },
+        "PPORT4": {
+            "icao": "PPORT4",
+            "name": "Port Four",
+            "rwy": {
+                "KMHT6" : [["TYLIR", "A30+"]],
+                "KMHT24": [["TYLIR", "A30+"]],
+                "KMHT17": [["TYLIR", "A30+"]],
+                "KMHT35": [["TYLIR", "A30+"]]
+            },
+            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"], ["COTEE", "A93+"]],
+            "exitPoints": {
+                "BAF":[["BAF", "A168"]],
+                "CCC":["JJIMY", ["YANTC", "A163+"], ["PPORT", "A170"]],
+                "NELIE":["JJIMY", ["NELIE", "A170"]]
+            },
+            "draw": [
+                ["TYLIR", "PPORT", "SHOEL", "RBELA","COTEE", "BAF*"],
+                ["TYLIR", "PPORT", "SHOEL", "RBELA","COTEE", "JJIMY", "YANTC", "CCC*"],
+                ["TYLIR", "PPORT", "SHOEL", "RBELA","COTEE", "JJIMY", "NELIE*"]
+            ]
+       }
+    },
+    "stars": {
+        "ROZZE2": {
+        "icao": "ROZZE2",
+        "name": "Rozze Two",
+        "entryPoints": {
+            "MUCOW": [["MUCOW", "A190-|S280"], ["FFATY", "A200-|A160+"], ["CRIBB", "A160-|A130+"], ["MONRC", "A90+"]],
```

Comment:
MUCOW is at or _above_ FL190, so it should read A190+.

Altitude restrictions should be written with the lower first and the higher last. so FFATY should be A160+|A200-

Reply:
✔️